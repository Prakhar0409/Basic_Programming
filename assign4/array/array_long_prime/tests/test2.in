10
1
2
6
4
5
3
7
2
11
13
