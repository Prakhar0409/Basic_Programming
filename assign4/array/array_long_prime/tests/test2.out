Enter number of elements 
Enter 10 integers 

5 3 7 2 11 13 
Start  =  4
Length =  6
